cmd/start-ssh-agent.cmd

```diff
@@ -73,6 +73,8 @@ if exist "%GIT%" (
         set "HOME=%USERPROFILE%"
         "!SSH_ADD!"
         echo.
+    ) else (
+        set "HOME=%USERPROFILE%"

```

Comment:
Bummer! I did not even look for local blocks, my bad. But you made me aware of another issue: should `HOME` not be set *before* line 61 so that `ssh-add` accesses the correct directory?